### PR TITLE
Remove years from copyright notes

### PR DESCRIPTION
**Pull request by s0me0ne-unkn0wn**
See https://github.com/paritytech/substrate/pull/13415#issuecomment-1435710397

### CODE REVIEW

File: file_header.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2017-2022 Parity Technologies (UK) Ltd.
+// Copyright 2017-2023 Parity Technologies (UK) Ltd.
```
Comment (ggwpez) [inline file_header.txt:1]:
```suggestion
// Copyright Parity Technologies (UK) Ltd.
```
See https://github.com/paritytech/substrate/pull/13415

Comment (s0me0ne-unkn0wn) [reply file_header.txt:1]: Even better!

**Comment by bkchr** [comment]
Can you please do this for all files?

**Review by bkchr — APPROVED** [review]

**Comment by s0me0ne-unkn0wn** [comment]
> Can you please do this for all files?

You mean for all the source files in the Polkadot repo? :scream_cat: 
No problem :)

**Review by liamaharon — APPROVED** [review]

**Review by ordian — APPROVED** [review]

**Comment by s0me0ne-unkn0wn** [comment]
bot merge

**Pull request merged**